test/fixtures/add-and-subtract.js

```diff
@@ -0,0 +1,15 @@
+const daysOfSun = 250,
+      islands = { inhabited: 170, total: 2000},
+      weddingEvents = ['plate-throwing','dancing' ];
```

Comment:
'weddingEvents' is assigned a value but never used. A space is required after ','. There should be no space before ']'.